components/onboarding_tasks/onboarding_tasks_manager.tsx

```diff
@@ -163,13 +244,24 @@ export const useTasksList = () => {
 export const useTasksListWithStatus = () => {
     const dataInDb = useSelector((state: GlobalState) => getCategory(state, OnboardingTaskCategory));
     const tasksList = useTasksList();
+    const getTaskDetails = useGetTaskDetails();
     return useMemo(() =>
         tasksList.map((task) => {
             const status = dataInDb.find((pref) => pref.name === task)?.value;
             return {
                 name: task,
                 status: status === FINISHED.toString(),
-                label: taskLabels[task],
+                label: () => {
+                    const {id, svg, message} = getTaskDetails[task];
+                    return (
+                        <Fragment key={id}>
+                            <picture>
+                                {svg}
```

Comment:
nit: As all svg will use same width/height, the component could be instanced here.